server/src/postgres.rs

```diff
@@ -66,9 +66,8 @@ impl SignalDatabase for PostgresDatabase {
             FROM
                 accounts
             WHERE
-                $1 = $2
+                aci = $1 OR pni = $1
```

Comment:
I have already made these changes for this file

Reply:
Fixed in merge conflicts